miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultLibraryDilutionService.java

```diff
@@ -204,7 +204,7 @@ private void loadChildEntities(LibraryDilution dilution) throws IOException {
     if (dilution.getLibrary() != null) {
       dilution.setLibrary(libraryDao.get(dilution.getLibrary().getId()));
     }
-    if (dilution.getTargetedSequencing() != null) {
+    if (dilution.getTargetedSequencing() != null && dilution.getTargetedSequencing().getId() != null) {
```

Comment:
I'll fix the root problem instead, which is that the DilutionDto->Dilution method was always creating a targeted sequencing and setting it on the dilution, even when the DilutionDto.targetedSequencingId was empty.